content/blog/convert-a-module-for-continuous-delivery.en.md

```diff
@@ -133,22 +133,45 @@ git checkout -b add-new-ci
 Suggest adding commits to this working branch as necessary when changing
 files in each section.
 
-## Change repository folder structure
+## Add CI pipeline configuration files
 
-We must change the folder structure because that is a prerequisites for
-the build and automatic release.
+Run the following to get the necessary pipeline files and folders that we
+need to covert the module.
+
+Overwrite any files it suggest, then check the `git` diff if you need to
+keep anything. Remove any .bak files that are created if they ar no longer
+needed.
+
+```powershell
+cd C:\source\{RepositoryName}
+
+Install-Module -Name 'Sampler' -Scope 'CurrentUser'
+$samplerModule = Import-Module -Name Sampler -PassThru
+
+$invokePlasterParameters = @{
+  TemplatePath = Join-Path -Path $samplerModule.ModuleBase -ChildPath 'Templates/Sampler'
+  DestinationPath = '..\'
+  ModuleType = 'dsccommunity'
+  ModuleName = Split-path -Leaf $PWD
+  SourceDirectory = 'source'
+}
+
+Invoke-Plaster @invokePlasterParameters
+```
 
->**NOTE:** The source folder could also have the same name as the module
+>**NOTE:** The `SourceDirectory` folder could also have the same name as the module
```

Comment:
As this default to source in the code above, we could potentially remove those details from here.

Reply:
Done